openbb_terminal/jupyter/reports/reports_controller.py

```diff
@@ -28,7 +28,7 @@ class ReportController(BaseController):
     reports_folder = os.path.dirname(os.path.abspath(__file__))
 
     report_names = [
-        notebooks.strip(".ipynb")
+        notebooks[:-6]
```

Comment:
not critical to me, but why?

Reply:
Took me some time to figure out that the strip was problematic. I had misunderstood it's usage. See
<img width="250" alt="Screenshot 2022-05-03 at 11 10 27" src="https://user-images.githubusercontent.com/25267873/166436453-3379647b-fc8b-42d4-b430-7f1942a7d50c.png">